components/rhel/rhel-ose/Vagrantfile

```diff
@@ -69,8 +76,19 @@ Vagrant.configure(2) do |config|
     sudo setsebool -P virt_sandbox_use_fusefs 1
   SHELL
 
+  # landrush
+  config.landrush.enabled = true
+  config.landrush.tld = TLD
+  config.landrush.host TLD, HOSTNAME
+  config.landrush.guest_redirect_dns = false
```

Comment:
@hferentschik yes. You are absolutely right about this line and I missed it during my testing :+1: